### PR TITLE
feat: new nlp pattern input

### PR DESCRIPTION
**Pull request by marrouchi**
# Motivation

This PR introduces a new react component for NLP pattern input.

# Type of change:

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added unit tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: frontend/src/components/visual-editor/BlockDialog.tsx [hunk 1]
```diff
@@ -69,24 +69,25 @@ const BlockDialog: FC<BlockDialogProps> = ({
...
+      typing: 0,
+      content: {
+        display: OutgoingMessageFormat.list,
+        top_element_style: "compact",
+        limit: 2,
       },
-      assign_labels: block?.assign_labels || [],
-      message: block?.message || [""],
+      assignTo: block?.options?.assignTo,
     },
```
Comment (marrouchi) [inline frontend/src/components/visual-editor/BlockDialog.tsx:85]: Can we add some defaults for context var capture and local fallback ?

File: frontend/src/components/visual-editor/form/inputs/triggers/PatternInput.tsx [hunk 2]
```diff
@@ -147,83 +145,14 @@ const PatternInput: FC<PatternInputProps> = ({
...
-                      id: entity.id,
-                      value: "any",
-                    } as INlpValue);
-                  }
-                }
-                acc.push(curr);
-
-                return acc;
-              }, [] as INlpValue[]);
+              setPattern(data);
```
Comment (marrouchi) [inline frontend/src/components/visual-editor/form/inputs/triggers/PatternInput.tsx:152]: If we will not use the event, let's remove it from the prop.

**Review by marrouchi — COMMENTED** [review]
Awesome @yassinedorbozgithub left some minor comments

File: frontend/src/components/visual-editor/BlockDialog.tsx (hunk 1 above)
Comment (yassinedorbozgithub) [reply frontend/src/components/visual-editor/BlockDialog.tsx:85]: Sure

**Comment by yassinedorbozgithub** [comment]
> Awesome @yassinedorbozgithub left some minor comments

All the comments are addressed :white_check_mark:

**Pull request merged**